src/lib_ccx/hardsubx_imgops.c

```diff
@@ -105,4 +105,50 @@ void rgb_to_lab(float R, float G, float B,float *L, float *a, float *b)
 	*b = YELLOW;
 }
 
+void rgb2lab(float R, float G, float B,float *L, float *a, float *b){
```

Comment:
Just use the old rgb_to_lab function, why create a duplicate?

Reply:
The old rgb_to_lab function doesn't cater the Russian News Tickers, creates a very dull image, I rewrote the function with little changes, change in threshold basically to get a great thresholded text

Reply:
Not good practice at all. You can do the same inside the russian function without creating a duplicate call which future contributors may confuse as standard.

Reply:
Noted :)